pkgs/development/python-modules/langchain-core/default.nix

```diff
@@ -83,15 +83,25 @@ buildPythonPackage rec {
   '';
 
   passthru = {
+    # Updates to core tend to drive updates in everything else
     updateScript = writeScript "update.sh" ''
       #!/usr/bin/env nix-shell
       #!nix-shell -i bash -p nix-update
 
       set -u -o pipefail +e
+      # Common core
       nix-update --commit --version-regex 'langchain-core==(.*)' python3Packages.langchain-core
       nix-update --commit --version-regex 'langchain-text-splitters==(.*)' python3Packages.langchain-text-splitters
       nix-update --commit --version-regex 'langchain==(.*)' python3Packages.langchain
       nix-update --commit --version-regex 'langchain-community==(.*)' python3Packages.langchain-community
+      nix-update --commit --version-regex 'langchain-text-splitters==(.*)' python3Packages.langchain-text-splitters
```

Comment:
```suggestion
```
duplicate

Reply:
@natsukium thank you. Where's the 🤦🏻 emoji when I need it?